.circleci/config.yml

```diff
@@ -0,0 +1,58 @@
+version: 2
+
+aliases:
+  - &pollsapi-image
+    image: circleci/python:2.7.15
+    environment:
+      DATABASE_URL: postgresql://root@localhost/circle_test?sslmode=disable
+      PORT: 8000
+
+  - &database-image
+    image: circleci/postgres:9.6.2
+    environment:
+      POSTGRES_USER: root
+      POSTGRES_DB: circle_test
+
+workflows:
+  version: 2
+
+  pollsapi:
+    jobs:
+      - test
+      - test-dredd
+
+jobs:
+  test:
+    docker:
+      - <<: *pollsapi-image
+      - <<: *database-image
+
+    steps:
+      - checkout
+      - run:
+          command: |
+            virtualenv venv
+            source venv/bin/activate
+            pip install -r requirements.txt
+            python manage.py migrate
+            python manage.py loaddata initial_data
+            python manage.py test
```

Comment:
Why are all these in a single run command?

Reply:
They all need to be ran from within the same virtualenv created at the start and sourced in second line so I grouped them up so they're executed in that env.

This gets a bit simpler with pipenv, and in following PR I will propose something the following as the `pipenv run` wrapper gives us easy way to get into the env.

```diff
diff --git a/.circleci/config.yml b/.circleci/config.yml
index 3543a59..c8e3aa9 100644
--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -13,6 +13,13 @@ aliases:
       POSTGRES_USER: root
       POSTGRES_DB: circle_test

+  - &bootstrap
+    run:
+      command: |
+        pipenv install
+        pipenv run python manage.py migrate
+        pipenv run python manage.py loaddata initial_data
+
 workflows:
   version: 2

@@ -29,14 +36,10 @@ jobs:

     steps:
       - checkout
+      - <<: *bootstrap
       - run:
           command: |
-            virtualenv venv
-            source venv/bin/activate
-            pip install -r requirements.txt
-            python manage.py migrate
-            python manage.py loaddata initial_data
-            python manage.py test
+            pipenv run python manage.py test

   test-dredd:
     docker:
@@ -48,11 +51,7 @@ jobs:
       - checkout
       - run:
           command: npm install dredd --no-optional
+      - <<: *bootstrap
       - run:
           command: |
-            virtualenv venv
-            source venv/bin/activate
-            pip install -r requirements.txt
-            python manage.py migrate
-            python manage.py loaddata initial_data
-            ./node_modules/.bin/dredd
+            pipenv run ./node_modules/.bin/dredd
```